docs/adminguide/src/site/pages/installation_upgrade/TdsBehindProxy.md

```diff
@@ -29,25 +39,109 @@ There are two methods to accomplish this:
+    Configuration for the Tomcat AJP connector (for use with Apache's `mod_jk`).
+  * [The Apache Tomcat Connectors: mod_jk, ISAPI redirector, NSAPI redirector](https://tomcat.apache.org/connectors-doc/index.html){:target="_blank"}
+    Apache JServ Protocol (AJP) Protocol reference, configurations and fine-tuning of a reverse proxy set up using the `mod_jk` Apache module.
+
+## Implementing The Tomcat-Apache Proxy Using HTTP
+
+The following example shows how to implement a reverse proxy using the Apache HTTP Server and the Tomcat Servlet Container using HTTP via `mod_proxy`/`mod_proxy_http`.
+
+### Configure Apache HTTP Server To Use `mod_proxy` and `mod_proxy_http` 
+
+1. Ensure your Apache HTTP Server installation includes the `mod_proxy` and `mod_proxy_http` modules.
+   
+   If you are uncertain, you can check to see if these modules are included either as a Dynamic Shared Object module or directly compiled into the `httpd` binary file by running the `apachectl` (a.k.a, Apache Control) command with the `-M` option:
+   
+      ~~~bash
+      # /usr/local/apache/bin/apachectl -M
+      Loaded Modules:
+       core_module (static)
+       ...
+       proxy_module (shared)
+       proxy_http_module (shared)
+       ...
+      ~~~
+
+2. Configure the reverse proxy in the Apache HTTP Server.
+
+   Utilize the `ProxyPass` and `ProxyPassReverse` directives to tell Apache to forward http://yourhost.edu/thredds/* to the Tomcat HTTP connector listening on port `8080` on `localhost`:
+
+   ~~~bash
+   ProxyPass         /thredds  http://localhost:8080/thredds
+   ProxyPassReverse  /thredds  http://localhost:8080/thredds
+   ~~~
+
+   Or if you are setting up a reverse proxy to a TDS hosted on a different server other than `localhost`:
+
+   ~~~bash
+   ProxyPass         /thredds  https://tds_host_location:8443/thredds
+   ProxyPassReverse  /thredds  https://tds_home_location:8443/thredds
+   ~~~
+
+   Notice the use of the HTTPS protocol and port in the above example.
+   
+   {%include important.html content="
+    We strongly encourage you to implement your reverse proxy over HTTPS instead of HTTP if you are proxying to a TDS instance installed on another host other than `localhost`.
     " %}
-    
+
+### Configure Tomcat For The proxy
+
+Configure the Tomcat HTTP Connector (or SSL/TLS HTTP/1.1 Connector) with the appropriate proxy settings:
+
+   ~~~bash
+   <Connector port="8080"
+              protocol="HTTP/1.1"
+              ...
+              proxyName="yourhost.edu"
+              proxyPort="80"/>
+   ~~~
+
 ## Implementing The Tomcat-Apache Proxy Using AJP
 
-The following example shows how to implement a proxy using the Apache HTTPD server, Tomcat Servlet Container, and Tomcat's `mod_jk` on a linux system.
+There are two options available for show how to implement a proxy using the Apache HTTP Server and the Tomcat Servlet Container via AJP:  using and `mod_proxy`/`mod_proxy_ajp` or `mod_jk`. 
```

Comment:
I think this needs just a little tweaking, wording-wise. Perhaps something like:

There are two options available for implementing a proxy using the Apache HTTP Server and the Tomcat Servlet Container via AJP: using `mod_proxy`/`mod_proxy_ajp`, or `mod_jk`.